packetbeat/_meta/beat.full.yml

```diff
@@ -432,23 +438,22 @@ packetbeat.protocols.nfs:
 #
 # Process matching is optional and can be enabled by uncommenting the following
 # lines.
-#
-#packetbeat.procs:
-#  enabled: false
-#  monitored:
-#    - process: mysqld
-#      cmdline_grep: mysqld
-#
-#    - process: pgsql
-#      cmdline_grep: postgres
-#
-#    - process: nginx
-#      cmdline_grep: nginx
-#
-#    - process: app
-#      cmdline_grep: gunicorn
+packetbeat.procs:
+  enabled: false
+  monitored:
+    - process: mysqld
+      cmdline_grep: mysqld
```

Comment:
Hmm, these are examples, kind of like we have for processors. I'd say it doesn't make sense to have them commented out. I know it's `enabled: false`, but still looks weird to me.